.github/workflows/test.yml

```diff
@@ -4,9 +4,7 @@ on:
   push:
     branches:
       - master
-      - develop
       - master-v0.x
-      - develop-v0.x
```

Comment:
This branch I would keep, at least until we complete v1

Reply:
How come? If we need to patch `0.28` we can do it directly on `master-v0.x` right?

Reply:
Maybe we could drop that branch the ?

Reply:
Honestly I thought we already had dropped it :smile: